addon/services/cookies.js

```diff
@@ -116,31 +114,34 @@ export default Ember.Service.extend({
   _filterCachedFastBootCookies(fastBootCookiesCache) {
     let { hostname, path: requestPath, protocol } = this.get('_fastBoot.request');
 
-    return _collection.reduce(fastBootCookiesCache, (acc, cookie, name) => {
```

Comment:
`return Ember.A(_collection).reduce(…`

should work here as well, using for … in with continue feels much worse than using reduce.